packages/components/psammead-caption/CHANGELOG.md

```diff
@@ -3,21 +3,21 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
-<<<<<<< HEAD
-| 1.0.1   | [PR#424](https://github.com/bbc/psammead/pull/424) Add Snyk badge to readme |
-| 1.0.0   | [PR#413](https://github.com/bbc/psammead/pull/413) Add support for different scripts typographies |
-| 0.3.5   | [PR#377](https://github.com/bbc/psammead/pull/377) Add knobs addon using psammead-storybook-helpers |
-| 0.3.4   | [PR#407](https://github.com/bbc/psammead/pull/407) Organise dependencies properly |
-| 0.3.3   | [PR#410](https://github.com/bbc/psammead/pull/410) Update left/right padding to match article design |
-| 0.3.2   | [PR#398](https://github.com/bbc/psammead/pull/398) Update dependencies |
-| 0.3.1   | [PR#393](https://github.com/bbc/psammead/pull/393) Remove padding from left/right of caption. |
-| 0.3.0   | [PR#365](https://github.com/bbc/psammead/pull/365) Remove background colour, update colour to Cloud Dark. Use font style italic. |
-| 0.2.1   | [PR#323](https://github.com/bbc/psammead/pull/323) Update storybook badge url |
-| 0.2.0   | [PR#318](https://github.com/BBC/psammead/pull/318) Update to new font face and family. |
-| 0.1.6   | [PR#323](https://github.com/BBC/psammead/pull/323) Update readme storybook badge |
-| 0.1.5   | [PR#264](https://github.com/BBC/psammead/pull/319) Resolving package-lock issues. |
-| 0.1.4   | [PR#264](https://github.com/BBC/psammead/pull/264) Resolving package-lock issues. |
-| 0.1.3   | [PR#245](https://github.com/BBC-News/psammead/pull/245) Ensures documentation consistent across component packages. |
-| 0.1.2   | [PR#231](https://github.com/BBC-News/psammead/pull/231) Add link to Storybook to README |
-| 0.1.1   | [PR#227](https://github.com/BBC-News/psammead/pull/227) Replace @bbc/gel-constants and @bbc/gel-foundations-styled-component with [@bbc/gel-foundations in Psammead](https://github.com/BBC-News/psammead/issues/226). |
+| 1.1.4 | [PR#446](https://github.com/bbc/psammead/pull/446) De-italicise text in italic tag |
```

Comment:
```suggestion
| 1.1.0 | [PR#446](https://github.com/bbc/psammead/pull/446) De-italicise text in italic tag |
```
You'll want to increment the Psammead Caption package version, not the top-level Psammead repo version.